user_guide/intro_company_structure_org_selector.rst

```diff
@@ -0,0 +1,198 @@
+    * Eastern California Stationary Phone Freelance Employees
+
+So, the system administrator usually creates such reflection of the organization’s hierarchy and defines what users 
+belong to them. *This affects the access and permissions granted to the users within the company*.
+ 
+For example, users assigned directly to the  Eastern California business unit may see all the orders, requests and 
+sales reported by any of the users of the unit and its child unit. Thus an office manager will see all the accounts 
+created in any direction and by any kind of employee.
+
+Users assigned to the child units may see only the accounts created within their unit - so, employees of the Stationary 
+Phone department will only see the items reported by members of their department, and won’t see items reported by 
+representatives of the Mobiles’ department. 
+
+In the same manner, the access may be limited for users assigned to the child-units of the next level, e.g. freelancer’s 
+may only see the information that has been reported or assigned to them directly, and office employees will see 
+information reported by any of their co-workers.
+
+This will protect information on one hand, and help to protect users from being overwhelmed with non-relevant 
+information. 
+
+Please refer to the :ref:`Access and permissions guide <user-guide-user-management-permissions-basic>` for more details.
+
+.. hint::
+
+    By the user name, you can see the name of the business unit, that *"owns"* the user. This is also defined by the 
+    system administrator and serves as additional information for other users.
+
+**At least one business unit must be defined in the system.**
+
+Organizations
+^^^^^^^^^^^^^
+
+In addition to the business unit hierarchy, OroCRM Enterprise Edition provides the ability to create several 
+Organizations. Organizations may be described as large business units, independent enough in their operations to 
+require a high degree of separation and customization. Organizations provide a much higher degree of separation - every 
+organization in OroCRM is basically a business in its own right. They might have their own hierarchies of business 
+units, use different languages and locales, or even the most common entities such as Contacts or Accounts may differ 
+from one organization to another. Data records that exists in one organization are normally not available to another.
+
+Usually, organizations are need when their real-world counterparts have their own P&L reports and balance sheets. For 
+the telecom in the example above, different organization could represent offices in each of the states. Another good 
+example for several organizations would be a franchisor that has several franchisees - the best way to represent each 
+of them in OroCRM is a separate organization. Even a freelance entrepreneur working with several different businesses 
+from his computer will benefit using a separate organization to represent each of them in OroCRM.
```

Comment:
Edit: Usually, organizations are required when their real-world counterparts have their own P&L reports and balance sheets. For instance, the telecom company in the example above may use different organizations to represent offices in each state. Likewise, a franchisor that oversees several franchisees could represent each of them in OroCRM using a separate organization. However, even a freelance entrepreneur working with several different businesses from his computer could use a separate organization to represent each of them in OroCRM.